app/screens/content-view-screen/content-view-screen.tsx

```diff
@@ -85,38 +107,80 @@ export class ContentViewScreen extends React.Component<ContentViewScreenProps, {
     }
   }
 
+  private handleWebViewMessage = (event: WebViewMessageEvent) => {
+    try {
+      const { action } = JSON.parse(event.nativeEvent?.data)
+      switch (action) {
+        case "MOUNTED":
+          this.webViewRef?.current?.injectJavaScript(`
+            ${
+            // HACK:
+            // Since As we cannot determine which iframe(s) contain LikeCoin button,
+            // so we loop through and post message to all iframes
+            ""}
+            for (let i=0; i < window.frames.length; i+=1) {
+              let frame = frames[i];
+              frame.postMessage({ action: "DISABLE_BUTTON" }, '${this.content.getConfig('LIKECOIN_BUTTON_BASE_URL')}');
+            }
+          `)
```

Comment:
I think we can run `document.querySelectorAll('iframe')`
and `.getAttribute('src')` to avoid directly accessing src as frame, bypassing SOP